### PR TITLE
Changed Server_URL to a constant

### PR DESCRIPTION
**Pull request by live-wire**

### CODE REVIEW

**Comment by nkzawa** [comment]
Thanks for your PR! Could you remove comment since we use git history for these purpose?

**Comment by nkzawa** [comment]
please squash your commits.

**Comment by live-wire** [comment]
Squashed the commits and formatted the message :+1:

File: app/src/main/java/com/github/nkzawa/socketio/androidchat/Constants.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.github.nkzawa.socketio.androidchat;
+
+/**
+ * Created by dbatheja on 6/4/2015.
```
Comment (nkzawa) [inline app/src/main/java/com/github/nkzawa/socketio/androidchat/Constants.java]: Could you remove this comment? Sorry for not explaining enough.

**Comment by live-wire** [comment]
Removed unnecessary Comment :+1:

**Comment by nkzawa** [comment]
Awesome, thx :smile:

**Pull request merged**